mock/src/test/java/com/tngtech/keycloakmock/impl/helper/TokenHelperTest.java

```diff
@@ -44,17 +48,17 @@
 
   @BeforeEach
   void setup() {
-    doReturn(CLIENT_ID).when(session).getClientId();
-    doReturn(SESSION_ID).when(session).getSessionId();
-    doReturn(NONCE).when(session).getNonce();
-    doReturn(USER).when(session).getUsername();
-    doReturn(ROLES).when(session).getRoles();
+    lenient().doReturn(CLIENT_ID).when(session).getClientId();
```

Comment:
In general: rather than making these lenient, I'd prefer if you moved this into a more specific method and call that in every test case that needs it.

In this specific case, as I said before, I would like to re-use the session logic even for the client credentials flow.